### DISCUSSION TITLE
nytt eller gammalt nettsidedesign?

### DISCUSSION THREAD

**Discussion by ArvidWedtstein** [IM-Databasen/IMDB#13] [Announcements]

**Comment by ArvidWedtstein** [comment]
Meiningar?

**Reply by Lartrax** [reply]
tror me ska bytta design snart

**Reply by ArvidWedtstein** [reply]
så heller mørkt/svart design?

**Reply by ArvidWedtstein** [reply]
tror me kanskje burde setta opp ein colorscheme, sånn at me ikkje begynne med 15 forskjellige fargar liksom

**Comment by ArvidWedtstein** [comment]
https://tailwindcomponents.com/component/3d-button-2 ?
